server/admin/leave.js

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 'use strict';
 
-var errors = require('../lib/errors.js');
+var errors = require('../../lib/errors.js');
```

Comment:
same functionality as before. just moved out of `server/index.js` and into its own file.